mozilla_version/maven.py

```diff
@@ -54,3 +54,21 @@ def _compare(self, other):
             return -1
         else:
             return 0
+
+    @property
+    def is_beta(self):
```

Comment:
Suggestion: I wonder if we should `is_nightly`. Maybe not worth adding it now.

Reply:
What would be the default value of `is_nightly`.  Happy to add it.  :)